docs/src/11ty/html-transform.js

```diff
@@ -0,0 +1,27 @@
+const cheerio = require('cheerio');
```

Comment:
I started using this for turning string into manipulatable dom https://www.npmjs.com/package/linkedom in the hopes it is faster. No need to change this PR but curious what you think of linkedom as an option for this type of work

Reply:
Linkedom looks interesting, I'll need to check it out. 

I didn't realize we were having speed problems with cheerio; cheerio is also supposed to be a fast, not-JSDOM kind of tool.

The difference seems more a matter of taste to me. Do we go with linkedom because the interface is closer to browser-native APIs? Or, do we go with cheerio, which is closer to jQuery and full of its own proprietary conveniences?